src/connectors/aws_inventory.py

```diff
@@ -95,14 +95,14 @@
     },
     {
         'type': 'str',
-        'name': 'accounts_identifier',
-        'title': "AWS Accounts Connection Identifier",
+        'name': 'accounts_connection_name',
+        'title': "AWS Accounts Table Name",
         'prompt': (
-            "The custom name for your AWS Accounts Connection, if you provided one. You must provide either an "
+            "The custom name for your AWS Accounts Connection Table. You must provide either an "
```

Comment:
This should read "The name for your AWS Account Connection Table. [...]" instead of "The custom name"